script/Common/00_00.txt

```diff
@@ -1800,7 +1810,7 @@ God doesn't share his power for no reason.
 [sha:c84fb98de05f42ed9db72124e412784ed8e24ad8]{
 -- Page 135, Offset 450.
 -- 今の君があるかぎり、その目は決して間違った結果は生まないでしょう」
-As long as you remain the way you are now, your eyes will never bring forth any wrong."
+As long as you remain the way you are now, your eyes will never cause any wrongdoing."
```

Comment:
Wrongdoing is a behaviour/action not a result. Maybe 'your eyes will never be a tool for wrongdoing'

Reply:
I'd rather revert then